docs/quickstart/includes/publish-dotnet.md

```diff
@@ -1,18 +1,20 @@
-1. Change to the folder containing the `.nupkg` file.
+1. Open a command prompt and change to the folder containing your NuGet package file.
```

Comment:
FYI, this other PR is modifying the same file, and might cause merge conflicts with this PR if it's merged first: https://github.com/NuGet/docs.microsoft.com-nuget/pull/2862/files

Reply:
Thanks for the heads up. That PR is from a related story - the articles are different, but they use the same five include files. I think those includes have already been reviewed, so I can just upload those when that PR is merged. I may make some tweaks.